components/formats-gpl/src/loci/formats/in/SDTReader.java

```diff
@@ -65,11 +76,6 @@
   protected boolean preLoad = true;
 
   /*
-   * Array to hold re-ordered data for all the timeBins in one channel
-   */
-  protected byte[] chanStore = null;
```

Comment:
I renamed it as dataStore for consistency with PQBinReader.java and because it's not really channel specific in any way. I don't feel strongly about it though. Happy to change back if preferred.